src/Components/LpBackground.css

```diff
@@ -0,0 +1,26 @@
+.background {
```

Comment:
generally you don't mix your css files in with your component files. I would create a separate folder structure for your css.

Reply:
Ok, so I have seen it where they have created a styles folder, is this what your talking about? If so where exactly would you place this? Do you have two, one for the pages file and one for the components file or just one; and if this is the case where would you place it?